tests/test_push_docker.py

```diff
@@ -6,7 +6,7 @@
 import json
 import pytest
 
-from pubtools._quay.push_docker import mod_entry_point
+from pubtools._quay.push_docker2 import mod_entry_point
```

Comment:
Could you please some description to commit msg?

Reply:
You mean for moving push_docker to push_docker2? There's a separate commit for that.